docs/source/changelog.rst

```diff
@@ -1,6 +1,9 @@
 =========
 Changelog
 =========
+**Future Release**
+    * Changes
+        * Rename LabelTimes.name to LabelTimes.label_name (:pr:`126`)
```

Comment:
Occasionally the featuretools changelog adds a "Breaking Changes" section (see v0.13.0 change notes for an example), I think we should add something similar here